### ISSUE TITLE
Developing with local `pyre_extensions`

### ISSUE THREAD

**Issue by theodoretliu** [facebook/pyre-check#299]
I've added the following to `__init__.py` in `pyre_extensions`
```python
_C = TypeVar("_C")
_D = TypeVar("_D")

class Drop(Generic[_C, _D]):
    pass
```
and 
```ocaml
let test_check_drop_operator context =
  let assert_type_errors = assert_type_errors ~context in
  (* let assert_default_type_errors = assert_default_type_errors ~context in *)
  assert_type_errors
    {|
      from typing import Generic
      from pyre_extensions import Drop, ListVariadic
      from typing_extensions import Literal

      Shape = ListVariadic("Shape")

      class A(Generic[Shape]): ...

      def f(x: A[Shape]) -> A[Drop[Shape, Literal[0]]]: ...
    |}
    []
``` 
to `analysis/test/integration/annotationTest.ml`. However, I'm still getting errors when running the tests:
```
Undefined import [21]: Could not find a name `Drop` defined in module `pyre_extensions`.
Undefined or invalid type [11]: Annotation `Drop` is not defined as a type.
```
which seems to indicate that the updates to `pyre_extensions` are not being detected or the wrong one is being referenced. How to properly configure this for local development?

**Comment by arthaud** [comment]
Hi @theodoretliu,

The integration tests do not use `./pyre_extensions/__init__.py` directly. They use a small portion of typeshed and pyre_extensions, defined directly in `test/test.ml`:
https://github.com/facebook/pyre-check/blob/master/test/test.ml#L1365

**Comment by arthaud** [comment]
Closing this since it looks like this is solved. Please reopen if necessary.

**Comment by theodoretliu** [comment]
Thank you so much!
